core/templates/dev/head/pages/exploration-editor-page/settings-tab/settings-tab.directive.html

```diff
@@ -36,39 +36,51 @@ <h3>Basic Settings</h3>
+                    </em>
+                  </span>
+                </div>
               </div>
             </div>
             <div class="form-group">
               <label for="explorationInitStateName" class="col-lg-2 col-md-2 col-sm-2 float-left">Name of first card</label>
               <div class="col-lg-10 col-md-10 col-sm-10">
-                <select id="explorationInitStateName" class="form-control protractor-test-initial-state-select" ng-model="$ctrl.explorationInitStateNameService.displayed" ng-change="$ctrl.saveExplorationInitStateName()" ng-options="name as name for name in $ctrl.stateNames track by name" width="85%">
-                </select>
-                <span class="form-text" style="font-size: smaller">
-                  <em>This is the first card the learner sees when playing your exploration.</em>
-                </span>
+                <div class="row">
+                  <select id="explorationInitStateName" class="form-control protractor-test-initial-state-select" ng-model="$ctrl.explorationInitStateNameService.displayed" ng-change="$ctrl.saveExplorationInitStateName()" ng-options="name as name for name in $ctrl.stateNames track by name" width="85%">
+                  </select>
+                </div>
+                <div class="row">
+                  <span class="form-text" style="font-size: smaller">
+                    <em>This is the first card the learner sees when playing your exploration.</em>
+                  </span>
+                </div>
               </div>
             </div>
             <div class="form-group">
               <label for="explorationTags" class="col-lg-2 col-md-2 col-sm-2 float-left">Tags</label>
               <div class="col-lg-10 col-md-10 col-sm-10">
-                <div ng-if="$ctrl.hasPageLoaded">
-                  <select2-dropdown item="$ctrl.explorationTagsService.displayed"
-                                    choices="$ctrl.explorationTagsService.displayed"
-                                    allow-multiple-choices="true"
-                                    invalid-search-term-message="Add a new tag (using lowercase letters and spaces)..."
-                                    new-choice-regex="<[::$ctrl.TAG_REGEX]>"
-                                    width="85%"
-                                    placeholder="Skills, concepts, topics..."
-                                    on-selection-change="$ctrl.saveExplorationTags()">
-                  </select2-dropdown>
+                <div class="row">
+                  <div ng-if="$ctrl.hasPageLoaded" style="width:100%;">
```

Comment:
Can we add `class="row"` in this div instead creating a new div?

Reply:
That doesn't work, and further changes are needed to achieve the look it has in the photos. In addition, it follows the convention of having a row-div for every input and the elements inside it. But, I can change it if you want to!

Reply:
Ok, so just wanted to know, merging this with the above `div` tag doesn't work?

Reply:
Yes. Mergin doesnt work